platform/ios/MGLMapView.mm

```diff
@@ -142,6 +142,7 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
     if (self && [self commonInit])
     {
         self.styleURL = nil;
+        self.accessToken = [MGLAccountManager accessToken];
```

Comment:
I mimicked [`[MGLMapView initWithFrame:]`](https://github.com/friedbunny/mapbox-gl-native/blob/initWithCoder-access-token/platform/ios/MGLMapView.mm#L113) — is there any difference here that we should be cautious of?